packages/sender/src/lib/sender.ts

```diff
@@ -85,7 +85,9 @@ const Sender: WalletBehaviourFactory<InjectedWallet> = async ({
   const setupEvents = () => {
     _state.wallet.on("accountChanged", async (newAccountId) => {
       logger.log("onAccountChange", newAccountId);
-      emitter.emit("signedOut", null);
+      emitter.emit("accountsChanged", {
+        accounts: [{ accountId: newAccountId }],
+      });
```

Comment:
Have you tested it, is changing accounts working properly?

I think the idea to emit `signedOut` event is because switching accounts did not work properly.

Import 2 different accounts in sender and try switching between the two.